core/src/main/java/org/stellar/anchor/config/ClientsConfig.java

```diff
@@ -98,7 +100,7 @@ private void validateClient(ClientConfig clientConfig, Errors errors) {
     }
   }
 
-  void validateCustodialClient(ClientConfig clientConfig, Errors errors) {
+  public void validateCustodialClient(ClientConfig clientConfig, Errors errors) {
```

Comment:
WDYT about validating all `destination_account`s are valid Stellar accounts? I don't think Anchors should discover this when a client makes a deposit.

Reply:
Agreed. I was thinking more along the lines of validating the format of the account ID using `KeyPair.fromAccountId`.

Reply:
If it's invalid account it wouldn't matter, as it's not going to pass account matching